Adds `hasLink` test for examples; update ~37 descriptions to pass (#474)
mapbox/mapbox-gl-js-docs: docs/pages/example/attribution-position.md

- "import html from './attribution-position.html';"
---

> This example places the [required](https://docs.mapbox.com/help/how-mapbox-works/attribution/) map attribution in the `top-left` position when initializing a map.
> 
> The application uses [`addControl`]https://docs.mapbox.com/mapbox-gl-js/api/map/#map#addcontrol with [`AttributionControl`](/mapbox-gl-js/api/markers/#attributioncontrol) to specify the position for this control as `top-left`.

Reviewer: ```suggestion
This example places the [required map attribution](https://docs.mapbox.com/help/how-mapbox-works/attribution/) in the `top-left` position of a map using [`addControl`](/mapbox-gl-js/api/map/#map#addcontrol) with [`AttributionControl`](/mapbox-gl-js/api/markers/#attributioncontrol).
```
